src/Components/test/testassets/ComponentsApp.App/Shared/MainLayout.cshtml

```diff
@@ -5,9 +5,6 @@
 </div>
 
 <div class="main">
-    @*<div class="top-row px-4">
-        <a href="http://blazor.net" target="_blank" class="ml-md-auto">About</a>
```

Comment:
I've been repointing this link to https://docs.microsoft.com/en-us/aspnet/core/razor-components in some templates/samples (e.g., Electron). But if you prefer to remove it that's fine too.

Reply:
I removed it because it's a test project and it's commented out. It's dead code.

Reply:
I commented it out at the time, it was causing me some grief as it was rendering in the middle of the page. I'm not sure why.